src/guides/en/understand-devtools/index.md

```diff
@@ -0,0 +1,236 @@
+
+![DevTools with the toolbar at the top, a panel in the middle, and a drawer at the bottom](./devtools-drawer.png)
+
+Let's zoom in one more level and focus on the content of the tool that's displayed in the panel. The following screenshot ignores the toolbar and drawer, and only shows the content of the panel where a tool is displayed. Each tool is different, and they don't all look the same, but the following screenshot shows the main elements that are often present in a tool: a sidebar on the left, with tabs used to display different sections of the tool, and a main content area on the right, which displays the content of the currently selected section:
+
+![A single DevTools tool, with a sidebar on the left, and a main content area on the right](./devtools-tool-with-sidear.png)
+
+Now let's put it all back together in one screenshot, so you can see how everything fits together. The screenshot shows the entire browser window, with a rendered webpage on the left, and DevTools on the right. DevTools has a toolbar, with tabs, a panel showing the selected tool, and a drawer area at the bottom. The panel shows a tool which contains a sidebar, and a main content area:
+
+![The browser with DevTools](./devtools-all-in-one.png)
+
+This might look like a lot of UI elements, and hard to understand at first. But, hopefully, with the way we've broken it down step by step, it helps making sense of the UI and forming a mental model of how it's organized.
+
+### Differences between browsers
+
+The UI of DevTools is something that each browser implements a little bit differently. The high-level UI description above is a generalization of the UI of DevTools across browsers.
+
+It's a good idea to test your website across different browsers, to find bugs that may manifest on certain browsers only. When this happens, you'll need to use the DevTools of the browser where the bug happens. Therefore, even if this browser isn't the one you normally use, it's good to be familiar with its DevTools.
+
+## The tools
+
+One of the aspects of DevTools that makes it complex is that it contains many different tools. The web development field is vast and there are many different tasks involved with it such as designing the look of a website, writing the code that makes it interactive, optimizing its performance, making sure it's accessible to everyone. DevTools contains tools that help with all of these tasks, and more. On top of this, DevTools accomodates for different types of users as well.
+
+This complexity leads to a high number of tools and features, which translates into a complex UI. But there are two things to keep in mind:
+
+* You don't need to use all the tools all of the time. You can focus on just a few tools that are relevant to the task you're working on.
+* You shouldn't be afraid to explore the tools that you don't know yet. You might find that they become useful to you at certain times.
```

Comment:
might be good to re-iterate that because nothing is persisted, you dont have to worry about things breaking and can always refresh and start over.

Reply:
Sounds good. Added.